house_of_refuge/frontend/src/components/ResourceList.js

```diff
@@ -127,7 +125,7 @@ export const ResourceList = ({initialResources, sub, subHandler, user, clearActi
                 .filter(r => statusFilter ? statusFilter.includes(r.resource) : true)
                 .filter(r => hotTopic ? isHot(r) : true)
                 .filter(r => turtleFilter ? r.turtle : true)
-                .filter(r => searchQuery ? resourceAsString(r).search(searchQuery) > -1 : true),
+                .filter(r => searchQuery ? ResourceAsStringForSearch(r).search(searchQuery) > -1 : true),
```

Comment:
Let's fix one minor bug if we touch those lines. `search` method allows the regex pattern in the argument. If the user types `+` then the app crashes.


```suggestion
                .filter(r => searchQuery ? ResourceAsStringForSearch(r).indexOf(searchQuery) > -1 : true),
```